### PR TITLE
Add Japanese to language switcher

### PR DESCRIPTION
**Pull request by spadgett**
We now have some translations for Japanese. Add the option to the language switcher.

/assign @rhamilto @rebeccaalpert

### CODE REVIEW

File: frontend/public/i18next-parser.config.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
+// TODO: Each language option should be displayed in its own native language instead of English.
 const SUPPORTED_LOCALES = {
   en: 'English',
+  ja: 'Japanese',
```
Comment (rhamilto) [inline frontend/public/i18next-parser.config.js:4]: This should be `ja: '日本語'`?  That's what the Customer Portal does.
Screen Shot 2020-11-11 at 2 08 26 PM

Comment (rebeccaalpert) [reply frontend/public/i18next-parser.config.js:4]: I agree with Robb! That's how it was originally in Joachim's PR as well.

Comment (spadgett) [reply frontend/public/i18next-parser.config.js]: Yeah, that was the TODO. I just didn't know the right value to use. Thanks! Updated.

**Review by rhamilto — COMMENTED** [review]
/lgtm

**Review by rebeccaalpert — APPROVED** [review]
Lgtm!

File: frontend/public/i18next-parser.config.js (hunk 1 above)
Comment (rebeccaalpert) [reply frontend/public/i18next-parser.config.js]: I know enough Japanese to confirm that value is correct. :)

**Pull request merged**